nncf/common/graph/transformations/commands.py

```diff
@@ -103,25 +105,25 @@ class TargetPoint:
     will be applied.
     """
 
-    def __init__(self, target_type: TargetType):
+    def __init__(self, target_type: TargetType, target_node_name: NNCFNodeName):
```

Comment:
I needed each target point to be formulated in terms of NNCFGraph for purposes unification, and it seems natural anyway that a target point in graph must have a reference to *some* object in graph. The object is node, and the reference is the node name.

Reply:
My concern is that the target point specifies the point in the model not in the NNCFGraph. If we commit that the NNCFNode.node_name is a 'layer/module' name,  we must make node_name a required parameter for NNCFNode.

Reply:
Made `node_name` a required parameter of `NNCFNode`.

Reply:
Am I right understand that NNCFNode.node_name == 'layer/module' name? How does it work with shared layers?

Reply:
A node corresponds to an execution of the operation. Instances of shared layer application have different node names (post-fixed with `^0`, `^1` etc. as usual), so that the "ignored_scopes", for example, could reference instances separately. The translation between a node name and the underlying layer - shared, custom or regular - is done at a lower, framework level.